_commands/basics/comm.md

```diff
@@ -0,0 +1,60 @@
+---
+---
+
+comm
+---
+Compares two sorted files line by line and writes three columns to standard output.
+<!-- one line explanation would go here -->
+
+<!-- minimal example -->
+~~~ bash
+# content of file1.txt
+$ cat file1.txt
+abc
+def
+ghi
+
+# content of file2.txt
+$ cat file2.txt
+abc
+ghi
+klm
+
+# compare file1.txt and file2.txt: first column showing unique lines of first file, 
```

Comment:
This explanation can go after the "`<!--more-->`" comment, to shorten the amount of information shown on the reference page.